scijava/scijava-ops-engine/src/main/java/org/scijava/ops/engine/struct/LazilyGeneratedFieldParameterData.java

```diff
@@ -26,7 +22,7 @@
  * {@code @param} and {@code @return} tags, the javadoc will be used to create
  * the parameter names and descriptions. Otherwise, reasonable defaults will be
  * used.
- * 
+ *
  * @author Gabriel Selzer
  */
 public class LazilyGeneratedFieldParameterData implements ParameterData {
```

Comment:
There's the potential for even more simplification here - I'm assuming that the YAML `OpInfo` pathway doesn't call this anymore (it shouldn't because we should get the data directly from the YAML) - so maybe the other pathways could just inline the useful logic from this class, and then we could delete this?

Reply:
Let's solve this as a part of scijava/scijava#138